config/site.yaml

```diff
@@ -4,5 +4,4 @@ author:
   email:
 metadata:
     description:
-taxonomies:
-  - track
\ No newline at end of file
+taxonomies: [track, category]
```

Comment:
@emb03 can you make sure the session collection based pages still work? I remember there was an issue for me when I passed an array to the taxonomies object like this.